x-pack/plugins/infra/public/test_utils/app_mock_renderer.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I added this wrapper to have it shared for all components and make it easier to test the components without wrapping them in each test - we can expand it with core services mocks if needed for other tests.

Reply:
This seems a common scenario, how are we handling it in other components' tests in the plugin? Is this the first time we need to wrap the tested component?

Reply:
I saw many different ways - mostly in the tests itself adding wrappers like `<I18nProvider>`, [CoreProvider](https://github.com/jennypavlova/kibana/blob/150893-infra-flyout-metadata-tests/x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/pod/pod_metrics_table.test.tsx/#L91) EuiThemeProvider ([as wrapper mostly in logs](https://github.com/jennypavlova/kibana/blob/150893-infra-flyout-metadata-tests/x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.test.tsx/#L30-L31)), etc.
Hmm, I was thinking that adding this will help us also in future tests but maybe we have many test cases requiring different wrappers so I will just add the wrapper in the test and remove this one.